packages/crypto/README.md

```diff
@@ -0,0 +1,7 @@
+# @turnkey/crypto
+
+This package consolidates some common cryptographic utilities used across our applications, particularly primitives related to keys, encryption, and decryption. For react-native you will need to polyfill our random byte generation by importing:
+
+```
+import 'react-native-get-random-values'
```

Comment:
could be a good idea to link to https://www.npmjs.com/package/react-native-get-random-values outright